docs/_partials/_customIssueFieldsKeyDeprecationNotice.mdx

```diff
@@ -0,0 +1,7 @@
+import { Admonition } from "@site/src/components/forDocs";
+
+<Admonition type="caution" title="Note">
+  We would like to inform you that the usage of the <b>"customIssueFields"</b>{" "}
```

Comment:
"We would like to inform you that"
do we need this statement ?
cant it be just "The usage of "customIssueFields"...."
and, instead of "key will be deprecated" , should it be "key has been deprecated" ?
since we have already deprecated this key.

Reply:
We got **Ack** from the product ( Gaurav ) for adding this note. Regarding the deprecation, it is not completely deprecated from webchat side, if customer is consuming **customIssueFields** key to pass CIF then it will on work for issue created via webchat, but won't work in case where issue created via Smart FAQ's ( aka Smart Forms )

@helpshift-gaurav could comment more on this.

Reply:
We want customer to slowly migrate from using `customIssueFields` to `cifs` , instead of directly notifying them that the `customIssueFields` are deprecated. Hence this note.